app/build.gradle.kts

```diff
@@ -73,6 +73,7 @@ android {
         unitTests {
             isIncludeAndroidResources = true
         }
+        // TODO: Convert it as a convention plugin once Flamingo goes out (b/242515648)
```

Comment:
Nit: Let's avoid links to bugs that aren't publicly accessible, we could link to the newly created GitHub issue instead.

Reply:
Done. Sgtm!